README.md

```diff
@@ -89,14 +110,17 @@ var app = express();
 
 Describe the differences between Express and Rails as backend frameworks.
 
-```text
+```
+Express uses a noSql db that can be easily used with Angular or other javascript languages without having to configure to other languages like ruby. Express uses Mongodb and Rails uses Active Record which is a Sql database. Both are great if you are sticking to a front end and back end that speaks the same language but can become complicated if not.
 
```

Comment:
careful here! active record is not exactly your database like mongodb.. what's important to note is that Rails is a Ruby-based framework and is set up to support relationships, while express is javascript and works well with non-relational dbs.